src/graphql/schema/uuid/abstract-repository/utils.ts

```diff
@@ -43,6 +44,29 @@ export function createRepositoryResolvers<
       if (!entity.currentRevisionId) return null
       return dataSources.serlo.getUuid<R>({ id: entity.currentRevisionId })
     },
+    async revisions(entity, cursorPayload, { dataSources }) {
+      const revsFromSerlo = await Promise.all(
```

Comment:
Not sure about the naming here. I don't see the advantage of the additional explicitness and would prefer consistency with how we named things in other places, i.e.:
```suggestion
      const revisions = await Promise.all(
```